addons/website_forum/static/src/js/website_forum.js

```diff
@@ -357,11 +357,20 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
                     }
                     $(elem).nextAll('.flag_validator').removeClass('d-none');
                 } else if (data.success === 'post_flagged_non_moderator') {
-                    const forumAnswer = elem.closest('.forum_answer');
                     elem.innerText = _t(' Flagged');
                     elem.prepend(child);
-                    forumAnswer.fadeIn(1000);
-                    forumAnswer.slideUp(1000);
+                    const postWrapper = elem.closest('.o_wforum_post_wrapper');
+                    if (postWrapper && postWrapper.classList.contains('o_wforum_answer')) {
+                        // Fade in and slide up the answer until the element's height is 0 at which point the element is hidden.
+                        postWrapper.style.transition = "opacity 0.5s linear 0s, height 0.6s ease-out 0s";
+                        postWrapper.style.opacity = 0;
+                        setTimeout(() => {
+                            postWrapper.style.height = postWrapper.clientHeight + 'px';
+                            postWrapper.offsetHeight;  // This forces a reflow so that the browser register the height for the transition
+                            postWrapper.style.height = 0;
+                        }, 500);
```

Comment:
Can we not add delay to the height transition instead ? ` height 0.6 ease-out 0.5s `

Also, any reason we need to read offsetHeight instead of just setting 
`postWrapper.style.height = postWrapper.clientHeight + 'px';` , starting the transition?

Reply:
Hello @Odoonan :slightly_smiling_face: 

I was reading offsetHeight so that the browser registers the newly assigned client height before starting the transition but actually setting a timeout does the trick and is much more understandable  :laughing: 

About the delay, as I need the timeout anyway I just set it to 0 in the css transition.

(I added some technical comment on the commit)

Cheers!